tests/fileTypeTests.py

```diff
@@ -110,7 +110,7 @@ def test_program_mixed(self):
         """Test mixed job with some rows failing."""
         jobId = self.jobIdDict["programMixed"]
         self.passed = self.run_test(
-            jobId, 200, "finished", 21060, 4, "complete", 118, True)
+            jobId, 200, "finished", 20998, 4, "complete", 118, True)
```

Comment:
Yes, good point! Because the SQL queries return the column names in the physical database, that's what is causing the new file sizes. One acceptance criteria on this story states that we should be returning the long names, so we won't close the book on this until a subsequent PR makes that change.

In the meantime, however, I thought it was more important to do the first step of getting the db columns updated so people can write queries using the new names.